apps/dapp-console/app/settings/components/LinkedWallet.tsx

```diff
@@ -25,12 +27,34 @@ export const LinkedWallet = ({ wallet, onUnlink }: LinkedWalletProps) => {
     [setIsDialogOpen],
   )
 
+  const handleCopy = useCallback(() => {
+    navigator.clipboard.writeText(wallet.address)
+    toast({
+      description: 'Address Copied',
+      duration: 1000,
+    })
+  }, [wallet, toast])
+
   return (
     <div className="flex flex-row gap-2">
       <Input value={wallet.address} disabled />
+      <Button
+        variant="secondary"
+        className="px-3"
```

Comment:
definitely dont do in this pr and should be a broader discussion with the team. but I wonder if we should make these tailwind classes constants, so if we ever do want to upgrade them across all of our design system, it is an easy change. cc @imranjami @tarunkhasnavis @jakim929

Reply:
i'm not convinced we actually will need to add padding, i was just seeing if it will look better or not without adding padding right now i think the buttons look like eggs

<img width="906" alt="Screenshot 2024-03-22 at 1 11 22 PM" src="https://github.com/ethereum-optimism/ecosystem/assets/1761993/22055d38-9a13-4b86-828e-d9cd297a5c3c">